.travis.yml

```diff
@@ -21,22 +21,28 @@ matrix:
       env: SYMFONY_VERSION=4.0.*
     - php: 7.2
       env: SYMFONY_VERSION=3.4.*
+    - php: 7.2
+      env:
+        - SYMFONY_VERSION=4.0.*
+        - FLYSYSTEM_VERSION=1.0.*
```

Comment:
i like the approach we took here: https://github.com/php-http/HttplugBundle/blob/f034b168dde627de027c3d874410b10df2aa82af/.travis.yml#L56 - we define one dependencies variable and if any is set, have composer install those.